index.bs

```diff
@@ -2338,10 +2341,116 @@ When this method is invoked, the user agent MUST execute the following algorithm
     user without [=user consent|consent=], this step MUST NOT be executed before |lifetimeTimer| has expired. See
     [[#sctn-assertion-privacy]] for details.
 
-During the above process, the user agent SHOULD show some UI to the user to guide them in the process of selecting and
-authorizing an authenticator with which to complete the operation.
 </div>
 
+#### <dfn for="PublicKeyCredential" algorithm="Issuing a credential request to an authenticator">Issuing a Credential Request to an Authenticator</dfn> #### {#sctn-issuing-cred-request-to-authenticator}
+
+This sub-algorithm of {{PublicKeyCredential/[[DiscoverFromExternalSource]]()}} encompasses the specific UI context-independent
+steps necessary for requesting a [=credential=] from a given [=authenticator=], using given {{CredentialRequestOptions}}. It is called by
+{{PublicKeyCredential/[[DiscoverFromExternalSource]]()}} from various points depending on which [=user mediation=] the
+present [=authentication ceremony=] is subject to (e.g.: {{CredentialMediationRequirement/conditional}} mediation).
+
+This algorithm accepts the following arguments:
+
+<dl dfn-type="argument" dfn-for="Issuing a credential request to an authenticator">
+    :   <dfn>authenticator</dfn>
+    ::  A [=client platform=]-specific handle identifying an [=authenticator=] presently available on this [=client platform=].
+
+    :   <dfn>savedCredentialIds</dfn>
+    ::  A [=map=] containing [=authenticator=] → [=credential ID=]. This argument will be modified in this algorithm.
+
+    :   <dfn>options</dfn>
+    ::  This argument is a {{CredentialRequestOptions}} object whose
+        <code>|options|.{{CredentialRequestOptions/publicKey}}</code> member contains a {{PublicKeyCredentialRequestOptions}}
+        object specifying the desired attributes of the [=public key credential=] to discover.
```

Comment:
This parameter is used below as if it is just the `PublicKeyCredentialRequestOptions` part, not the whole `CredentialRequestOptions` object. There is a related, pre-existing issue in the parent `get()` (and `create()`) algorithms too, see: #1752

Reply:
Alright, let's fix it here only for this sub-algorithm in this PR to avoid piling up more changes on this. I had the caller pass in the `options.publicKey` instead.